<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite name="com.qtpselenium.hybrid.testcases.LoginTest" tests="2" time="29.033" ignored="0" failures="2" timestamp="23 Sep 2018 15:30:37 GMT" skipped="0" hostname="DESKTOP-NM05SVP" errors="0">
  <testcase name="doLogin" time="20.094" classname="com.qtpselenium.hybrid.testcases.LoginTest">
    <failure type="java.lang.AssertionError" message="Element not found password_id">
      <![CDATA[java.lang.AssertionError: Element not found password_id
	at org.testng.Assert.fail(Assert.java:94)
	at com.qtpselenium.hybrid.keywords.GenericKeywords.reportFailure(GenericKeywords.java:166)
	at com.qtpselenium.hybrid.keywords.GenericKeywords.getObject(GenericKeywords.java:153)
	at com.qtpselenium.hybrid.keywords.GenericKeywords.type(GenericKeywords.java:82)
	at com.qtpselenium.hybrid.driver.DriverScript.executeTest(DriverScript.java:56)
	at com.qtpselenium.hybrid.testcases.LoginTest.doLogin(LoginTest.java:51)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:483)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:851)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1177)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:756)
	at org.testng.TestRunner.run(TestRunner.java:610)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:387)
	at org.testng.SuiteRunner.access$000(SuiteRunner.java:39)
	at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:421)
	at org.testng.internal.thread.ThreadUtil$2.call(ThreadUtil.java:64)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
]]>
    </failure>
  </testcase> <!-- doLogin -->
  <testcase name="doLogin" time="8.939" classname="com.qtpselenium.hybrid.testcases.LoginTest">
    <failure type="java.lang.AssertionError" message="Element not found password_id">
      <![CDATA[java.lang.AssertionError: Element not found password_id
	at org.testng.Assert.fail(Assert.java:94)
	at com.qtpselenium.hybrid.keywords.GenericKeywords.reportFailure(GenericKeywords.java:166)
	at com.qtpselenium.hybrid.keywords.GenericKeywords.getObject(GenericKeywords.java:153)
	at com.qtpselenium.hybrid.keywords.GenericKeywords.type(GenericKeywords.java:82)
	at com.qtpselenium.hybrid.driver.DriverScript.executeTest(DriverScript.java:56)
	at com.qtpselenium.hybrid.testcases.LoginTest.doLogin(LoginTest.java:51)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:483)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:851)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1177)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:756)
	at org.testng.TestRunner.run(TestRunner.java:610)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:387)
	at org.testng.SuiteRunner.access$000(SuiteRunner.java:39)
	at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:421)
	at org.testng.internal.thread.ThreadUtil$2.call(ThreadUtil.java:64)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
]]>
    </failure>
  </testcase> <!-- doLogin -->
</testsuite> <!-- com.qtpselenium.hybrid.testcases.LoginTest -->
